x-pack/solutions/observability/plugins/observability_onboarding/scripts/test/e2e.js

```diff
@@ -80,7 +80,6 @@ function runTests() {
     env: {
       ...process.env,
       CYPRESS_CLI_ARGS: JSON.stringify(cypressCliArgs),
```

Comment:
@mykolaharmash is Cypress still used here or are these configs remnants of the refactor?

Reply:
(just to add to the comment, this is present in main as well)

Reply:
oh, thank you for pointing out, it's a leftover, did not notice it during the refactor. I will create a PR to clean this up.